fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
So kibana is logging in JSONL format. Does fluentd detect that and handle
it differently? I would have expected fluentd to just pass it through as a
string. Or is the extra stuff coming from elasticsearch analysis of the
message?

On Wed, Sep 30, 2015 at 11:02 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In fluentd/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/4#discussion_r40804852>
> :
>
> > @@ -135,18 +134,36 @@
> >      bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
> >      ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
> >    </filter>
> > +  <filter **kibana**>
>
> I think the issue is how Kibana writes into its container logs.
> {"log":"{\"name\":\"Kibana\",\"hostname\":\"kibana-1-1lezg\",\"pid\":8,\"level\":30,\"msg\":\"No
> existing kibana index
> found\",\"time\":\"2015-09-29T20:04:06.713Z\",\"v\":0}\n","stream":"stdout","time":"2015-09-29T20:04:06.716406777Z"}
>
> It has a bunch of extra fields in there that we don't see with the other
> containers.
> ES: {"log":"[2015-09-29 19:43:28,333][INFO ][node ] [Vashti] initializing
> ...\n","stream":"stdout","time":"2015-09-29T19:43:28.334237592Z"}
>
> Fluentd: {"log":"2015-09-29 19:57:01 +0000 [info]: reading config file
> path=\"/etc/fluent/fluent.conf\"\n","stream":"stdout","time":"2015-09-29T19:57:01.143011924Z"}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/4/files#r40804852>
> .
>